Cargo.lock

```diff
@@ -1674,9 +1673,9 @@ checksum = "9a3a5bfb195931eeb336b2a7b4d761daec841b97f947d34394601737a7bba5e4"
 
 [[package]]
 name = "hyper"
-version = "0.14.20"
+version = "0.14.23"
```

Comment:
this one is important as it ships a fix for a nasty bug in the http2 client https://github.com/hyperium/hyper/releases/tag/v0.14.23